Services/services.h

```diff
@@ -72,7 +73,51 @@ typedef enum { TM_HK_PARAMETERS_REPORT = 0 } Housekeeping_Subtype;
 
 // COMMUNICATION SERVICE
 #define TC_COMMUNICATION_SERVICE 10
-typedef enum { GET_TEMP = 0 } Communication_Subtype;
+typedef enum {
+    S_GET_TEMP = 0,
+    S_GET_FREQ = 1,
+    S_GET_CONTROL = 2,
+    S_GET_ENCODER = 3,
+    S_GET_PA_POWER = 4,
+    S_GET_STATUS = 5,
+    S_GET_TR = 6,
+    S_GET_BUFFER = 7,
+    S_GET_HK = 8,
+    S_SOFT_RESET = 9,
+    S_GET_FULL_STATUS = 10,
+    S_SET_FREQ = 11,
+    S_SET_CONTROL = 12,
+    S_SET_ENCODER = 13,
+    S_SET_PA_POWER = 14,
+    S_GET_CONFIG = 15,
+    S_SET_CONFIG = 16
+} Sband_Subtype;
+
+typedef enum {
+    UHF_SET_STAT_CONTROL = 20,
+    UHF_SET_FREQ,
+    UHF_SET_PIPE_TIMEOUT,
+    UHF_SET_BEACON_T,
+    UHF_SET_AUDIO_T,
+    UHF_SET_PARAMS, //Merge last 4
+    UHF_RESTORE_DEFAULT,
+    UHF_LOW_PWR,
+    UHF_SET_DEST,
+    UHF_SET_SRC,
+    UHF_SET_MORSE,
+    UHF_SET_MIDI,
+    UHF_SET_BEACON_MSG,
+    UHF_SET_I2C,
+    UHF_WRITE_FRAM,
+    UHF_SECURE,
+    UHF_GET_FULL_STAT, // 0-244,249,250
+    UHF_GET_CALL_SIGN, // 245-246
+    UHF_GET_MORSE,
+    UHF_GET_MIDI,
+    UHF_GET_BEACON_MSG,
+    UHF_GET_FRAM,
+    //UHF_NEW_CMD, //not in here
```

Comment:
This `UHF_NEW_CMD` is a super interesting but also complicated feature from UHF EH. It basically allows you to define a new command. Based on my conversation with Thomas, I don't think it becomes necessary as a service. (Also tbh it'll be super difficult to implement it in GS and here)

Reply:
Ok, I expect it's probably something we can use on the ground before the mission

Reply:
True. So, you mean there's no need for it to be a subservice or should I work on it?

Reply:
I mean that it's probably fine that it's not available from the ground - don't worry about it.

Reply:
Is this the generic I2C command one? If it is then I would say don't include it as a subservice since the only situation in which we would need to use it is when the OBC is borked anyways